CondFormats/SiStripObjects/src/SiStripDetSummary.cc

```diff
@@ -34,7 +34,7 @@ void SiStripDetSummary::add(DetId detid, float value) {
       break;
   }
   detNum += layer * 10 + stereo;
-  // string name( detector + boost::lexical_cast<string>(layer) + boost::lexical_cast<string>(stereo) );
+  // string name( detector + std::to_string(layer) + std::to_string(stereo) );
```

Comment:
is this commented out line needed?

Reply:
I was doubtful there if needed by other developers. Will remove it if its of no use

Reply:
@mmusich @pieterdavid please comment

Reply:
it serves no purpose I can think of. Please remove.